.github/workflows/lib-codeql.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+      uses: actions/checkout@9a9194f87191a7e9055e3e9b95b8cfb13023bb08 # v4
```

Comment:
@mythi The same question, is the `6929` sha required or can it be changed to `v4` tag instead?

Reply:
I'm planning on running `frizbee actions ./github/workflows` once the tags are correctly updated (dependabot cannot do that so that's one last manual step.